packages/keystone/apolloServerPlugins/queryBlocking/constants.js

```diff
@@ -0,0 +1,5 @@
+const ERROR_TYPE = 'FORBIDDEN_REQUEST'
```

Comment:
It's better to move it inside `packages/keystone/apolloServerPlugins/queryBlocking/plugin.js` it used only there ...

Reply:
its also better to use `FORBIDDEN_REQUEST` const name for better readability

Reply:
inlined variable inside plugin.js